### PR TITLE
Minor change to workaround globalize bug

### PR DESCRIPTION
**Pull request by k3rn31**
This is the only way I found to workaround a bug introduced by
globalize/globalize on spree_contrib/spree_globalize. See pull request
'#6677' for more.

### CODE REVIEW

File: backend/app/controllers/spree/admin/resource_controller.rb [hunk 1]
```diff
@@ -158,7 +158,8 @@ def parent_data
   def parent
     if parent_data.present?
       @parent ||= parent_data[:model_class].
-          send("find_by_#{parent_data[:find_by]}", params["#{resource.model_name}_id"])
+        # Don't use find_by to workaround globalize/globalize#441 bug
+        send("find_by", parent_data[:find_by].to_s => params["#{resource.model_name}_id"])
```
Comment (mauazua) [inline backend/app/controllers/spree/admin/resource_controller.rb]: hi, @k3rn31 in your comment you specify that `find_by` should not be used but in the line below you use it anyway. Are you sure that this is ok? Besides that, could you please pass the first argument to `send` as a symbol?

**Review by mauazua — CHANGES_REQUESTED** [review]

**Comment by mauazua** [comment]
hi, @k3rn31. Thanks for submitting this pull request. Could you please tell me, in what case did you encounter this issue again?

**Comment by k3rn31** [comment]
Hi Marta, thanks for considering my PR.
The issue is due to the "globalize/globalize" gem which is included in "spree_contrib/globalize". The bug (globalize/globalize #423) breaks 'find_by_attribute_name' (sorry I miss-typed the comment, I will fix it in the next commit) raising a "NoMethodError in Spree::Admin::ProductPropertiesController#index" when you open  "/admin/products/example-product/product_properties".
You can take a look at the merged PR #6677 fixing the same problem in another part of spree.
I hope this explains it all.

File: backend/app/controllers/spree/admin/resource_controller.rb (hunk 1 above)
Comment (k3rn31) [inline backend/app/controllers/spree/admin/resource_controller.rb]: Hi i fixed the comment and passed the first argument to send as symbol as requested. Let me know if it is ok now. Thanks.

**Comment by mauazua** [comment]
hi, @k3rn31, thanks for the explanation. I think it's all ok now, but please squash your commits into one :)

**Comment by k3rn31** [comment]
Ok Marta, the commits are squashed together.

**Review by mauazua — APPROVED** [review]

**Pull request merged**